influxdb/influxdb.go

```diff
@@ -0,0 +1,36 @@
+package influxdb
+
+import (
+	"context"
+	"os"
+	"time"
+
+	influxdb2 "github.com/influxdata/influxdb-client-go"
+)
+
+func WriteData(serviceName string, containerID string, hostName string, servicePort string) {
```

Comment:
How about making a `struct{}` instead of declaring the variable alone?  For now, having these up to 4 variable seems okay, but we could easily leverage the `struct{}` and do not depends on changing the function parameters, instead we could just change the `struct` to add a new variable and pass as memory pointer; what do you think?

To illustrate the argument, the code will become something like:


```go
// file: influxdb/types.go

type ServiceMetrics struct {
        serviceName string
        containerID   string
        hostName     string
        servicePort   string
}
```
Note: the the struct begins with capital later, so its public expose from the package

the function signature will become:

```
func WriteData(metrics *ServiceMetrics)
````

On the function that will call this guy, we'll have something like:

```go
metrics := &influxdb.Metics{
     ....
}

influxdb.WriteData(metrics)
```

I should note, this is just an example, perhaps a better name should be considered :) 

If we want, later to add more metrics, we can easily extent the `struct{}` without having to change the function signature. What do you think?